client/app/recommendationListEntry.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import styles from './recommendationListEntry.css';
```

Comment:
Could use object destructuring here so each className below doesn't need to have ```style.``` in front

Reply:
I didn't know you could do that with css modules, thanks for the tip!